src/containers/LandingPage.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import LoginForm from '../components/LoginForm.jsx';
+import SignupForm from '../components/SignupForm.jsx';
+
+const LandingPage = () => (
+  <div>
+    <div>
+      LoginForm
+      <LoginForm />
+    </div>
+    <br />
+    <div>
+      SignupForm
+      <SignupForm />
```

Comment:
Will we be using LoginForm and SIgnUpForm as headers? can we wrap them with tags?